src/Mvc/Mvc.NewtonsoftJson/test/NewtonsoftJsonInputFormatterTest.cs

```diff
@@ -235,6 +235,45 @@ public Task ReadAsync_WithInputThatDeserializesToNull_SetsModelOnlyIfAllowingEmp
             Assert.Equal(expectedMessage, modelError.ErrorMessage);
         }
 
+        [Theory]
+        [InlineData("[5,7,3]", 0)]
+        [InlineData("[5, 'seven', 3]", 1)]
+        [InlineData("[5, 'seven', 3, 'notnum']", 2)]
+        public async Task ReadAsync_AllowMultipleErrors(string content, int failCount)
+        {
+            // Arrange
+            var failTotal = 0;
+            var serializerSettings = new JsonSerializerSettings
+            {
+                Error = delegate (object sender, ErrorEventArgs args)
+                {
+                    args.ErrorContext.Handled = true;
```

Comment:
I don't think you need to do this. The formatter should already be doing this on your behalf.